paf-mvp-frontend/src/utils/queue.ts

```diff
@@ -0,0 +1,64 @@
+import { Log } from '@core/log';
+
+const log = new Log('PAF', '#3bb8c3');
+
+// API for loading PAF asynchronously and run functions regardless
+// whether it gets loaded before or after their script executes.
+//
+// <script src="url/to/PAF-lib.js" async></script>
+// <script>
+//  var PAF = PAF || {};
+//  PAF.queue = PAF.queue || [];
+//  PAF.queue.push(functionToExecuteOncePrebidLoads);
+// </script>
+//
+// In asynchronous code:
+//  export const queue: CommandQueue = window.PAF.queue || [];
+//  window.PAF.queue = processCommands(queue);
+
+/** An operation executed asynchronously after the PAF-lib is loaded. */
+export type Command = () => void;
+
+/** Interface for processing every pushed commands as soon as possible.  */
+export interface ICommandProcessor {
```

Comment:
Why use an interface in this particular case? Just the `CommandProcessor` class would be enough, as the interface is exported but not used elsewhere.